primitives/runtime/src/traits.rs

```diff
@@ -1335,26 +1335,29 @@ pub trait GetNodeBlockType {
 	type NodeBlock: self::Block;
 }
 
-/// Something that can validate unsigned extrinsics for the transaction pool.
+/// Provide validation for unsigned extrinsics.
 ///
-/// Note that any checks done here are only used for determining the validity of
-/// the transaction for the transaction pool.
-/// During block execution phase one need to perform the same checks anyway,
-/// since this function is not being called.
+/// This trait provides two functions [`pre_dispatch`](Self::pre_dispatch) and
+/// [`validate_unsigned`](Self::validate_unsigned). The first one being called right before
+/// dispatching the call wrapped by an unsigned extrinsic. The second one mainly being used in the
+/// context of the transaction pool to check the validity of the call wrapped by an unsigned
+/// extrinsic.
 pub trait ValidateUnsigned {
 	/// The call to validate
 	type Call;
 
 	/// Validate the call right before dispatch.
 	///
 	/// This method should be used to prevent transactions already in the pool
```

Comment:
```suggestion
	/// The [`validate_unsigned`](Self::validate_unsigned)) method should be used to prevent transactions already in the pool
```
Inserting the parenthetical i.e. clause makes this tricky to read.

Reply:
I mean these docs are directly above the method I'm speaking about. I think it is fine to reference it by "this"